package.json

```diff
@@ -175,6 +179,8 @@
     "docs:api": "node ./scripts/docs/build-api-docs.js",
     "docs:serve": "http-server ./build/docs/",
     "docs:verify": "node ./scripts/web-crawler/index.js --docs",
+    "docs:lint": "node scripts/vale/vale.js",
+    "docs:vale": "vale",
```

Comment:
Why do we have both scripts here?

Reply:
`"docs:vale"` is used internally by `"docs:lint"`, as it otherwise cannot access the `vale` script. Additionally it allows executing `vale` without our wrapper, if anyone has the need for direct access.